falcon-taz/src/main/java/pt/haslab/taz/TraceProcessor.java

```diff
@@ -537,27 +537,27 @@ private void parseMessageHandlers()
 
             // Use a fast and a slow iterator to handle nested message handlers while iterating through the thread events.
```

Comment:
This comment is now outdated.

Reply:
Fixed in commit da28e4bde8df891167fb5590977c0fe30b8af086